unstructured_api_tools/pipelines/templates/pipeline_api.txt

```diff
@@ -28,12 +29,35 @@ async def pipeline_1(request: Request, file: UploadFile = File(),
 {% for param in multi_string_param_names %}
 {{param}}: List[str] = Form(default=[]),
 {% endfor %}
+{% if response_type %}
+response_type: str = "{{response_type}}",
+{% endif %}
 ):
 
     text = file.file.read().decode("utf-8")
     response = pipeline_api(text, {% for param in multi_string_param_names %}{{param}}, {% endfor %})
```

Comment:
Don't we need to pass in the response type here to get back the appropriate response?